Submitted By:     Nagy Bence <nagybence@tipogral.hu>
Date:             2005.02.03
Initial Packages: 2.9.4
Description:      This patch adds a new feature to makepkg.
                  If available, use ccache, instead of real gcc, g++, etc.

diff -Naur pacman-2.9.4.orig/scripts/makepkg pacman-2.9.4/scripts/makepkg
--- pacman-2.9.4.orig/scripts/makepkg   2005-01-07 00:53:27.000000000 +0100
+++ pacman-2.9.4/scripts/makepkg        2005-01-07 00:56:05.051397736 +0100
@@ -584,6 +584,9 @@
        exit 0
 fi

+# using ccache if exist
+[ -d /usr/lib/ccache/bin ] && PATH=/usr/lib/ccache/bin:$PATH
+
 # build
 msg "Starting build()..."
 build 2>&1
